client/my-sites/customer-home/locations/primary/index.jsx

```diff
@@ -11,19 +12,28 @@ import Webinars from 'my-sites/customer-home/cards/tasks/webinars';
 import FindDomain from 'my-sites/customer-home/cards/tasks/find-domain';
 import SiteSetupList from 'my-sites/customer-home/cards/tasks/site-setup-list';
 import { PerformanceTrackerStop } from 'lib/performance-tracking';
+import GoMobile from 'my-sites/customer-home/cards/features/go-mobile';
```

Comment:
As pointed out in https://github.com/Automattic/wp-calypso/issues/41880#issuecomment-626613499, we should create a new component that renders the generic `Task` component. You can see some examples in https://github.com/Automattic/wp-calypso/blob/ecc977eee5ecceea69ed86d39840a5273814a7d3/client/my-sites/customer-home/cards/tasks/connect-accounts/index.jsx or https://github.com/Automattic/wp-calypso/blob/ecc977eee5ecceea69ed86d39840a5273814a7d3/client/my-sites/customer-home/cards/tasks/find-domain/index.jsx.

Reply:
That was only a test for me to understand the mechanics of `Primary` and other components. I wasn't expecting a code review as I put `WP` 😁

Reply:
Gotcha! I thought WP was referring to WordPress 😅 We use the WIP nomenclature for work-in-progress PRs, so I updated the title 😉

Reply:
Now it is fine @mmtr